x-pack/solutions/security/plugins/security_solution/server/endpoint/lib/scripts_library/constants.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+const SCRIPTS_LIBRARY_FILE_INDEX_NAME_PREFIX = '.endpoint-script-file';
+
+export const SCRIPTS_LIBRARY_FILE_METADATA_INDEX_NAME_PREFIX = `${SCRIPTS_LIBRARY_FILE_INDEX_NAME_PREFIX}-meta`;
+export const SCRIPTS_LIBRARY_FILE_DATA_INDEX_NAME_PREFIX = `${SCRIPTS_LIBRARY_FILE_INDEX_NAME_PREFIX}-data`;
+
+export const SCRIPTS_LIBRARY_FILE_METADATA_INDEX_NAME = `${SCRIPTS_LIBRARY_FILE_METADATA_INDEX_NAME_PREFIX}-default`;
+export const SCRIPTS_LIBRARY_FILE_DATA_INDEX_NAME = `${SCRIPTS_LIBRARY_FILE_DATA_INDEX_NAME_PREFIX}-default`;
```

Comment:
Is the script lib going to be namespaced? I imagine the answer is no, leading to the fixed `-default`. But I don't know if its even necessary to do this convention.